<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hmi</class>
 <widget class="QWidget" name="hmi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>HMI</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Controller IP Address &amp; Port:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Controller_IP_Addr_Edit">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>34</y>
     <width>108</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>192.168.10.120</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Connect_Controller_Btn">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>65</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Controle Mode</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>System Status</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>201</width>
     <height>171</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Normal&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>INITIALIZE</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Connect_Controller_Status_Edit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>66</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>No</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Exit_Btn">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>370</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>571</width>
     <height>351</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Control Settings</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Point</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Update_Setpoint_Edit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Run_Control_Btn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Run Control</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Stop_Control_Btn">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop Control</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Update_Control_Config_Btn">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>260</y>
       <width>181</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Update Control Configurations</string>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>321</width>
       <height>231</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>PID Gains</string>
      </attribute>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>13</y>
         <width>54</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Kp</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>43</y>
         <width>54</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Ki</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>73</y>
         <width>54</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Kd</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>13</y>
         <width>113</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>1.0</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>43</y>
         <width>113</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>1.0</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_7">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>73</y>
         <width>113</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>1.0</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>10</y>
         <width>91</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Output Range:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>40</y>
         <width>54</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>High</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>70</y>
         <width>54</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Low</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_8">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>40</y>
         <width>61</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>100</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_9">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>70</y>
         <width>61</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_10">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>166</y>
         <width>113</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>0.001</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>136</y>
         <width>54</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>beta</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>106</y>
         <width>54</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>alpha</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_11">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>136</y>
         <width>113</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>1.0</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>166</y>
         <width>54</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>dt</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_12">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>106</y>
         <width>113</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>1.0</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>140</y>
         <width>82</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Auto?</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Process Data</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>System Monitoring</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QLineEdit" name="Controller_IP_Port_Edit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>34</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>60000</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
